### ISSUE TITLE
No module named 'cv2.cv2'

### ISSUE THREAD

**Issue by 112kutiko** [skvark/opencv-python#186]
i run test code 

get error

```
ModuleNotFoundError                       Traceback (most recent call last)
<ipython-input-8-950f128db003> in <module>
      1 import numpy as np
----> 2 import cv2
      3 
      4 cap = cv2.VideoCapture(0)
      5 

~\AppData\Roaming\Python\Python37\site-packages\cv2\__init__.py in <module>
      1 import importlib
      2 
----> 3 from .cv2 import *
      4 from .data import *
      5 

ModuleNotFoundError: No module named 'cv2.cv2'
```

 my code 
```
import numpy as np
import cv2

cap = cv2.VideoCapture(0)

while(True):
    # Capture frame-by-frame
    ret, frame = cap.read()

    if cap.isOpened():
        print("Webcam online.")

# When everything done, release the capture
cap.release()
cv2.destroyAllWindows()
```

my os win10
- 64 but i using anaconda 32
- opencv-python version  (4.0.0.21)

**Comment by skvark** [comment]
Sounds like you have a conflicting / another cv2 installation somewhere in your current environment. Create a fresh virtualenv and try to install ``opencv-python`` there. Also check that there is a cv2 .pyd file at ``~\AppData\Roaming\Python\Python37\site-packages\cv2``.

**Comment by 112kutiko** [comment]
> Sounds like you have a conflicting / another cv2 installation somewhere in your current environment. Create a fresh virtualenv and try to install `opencv-python` there. Also check that there is a cv2 .pyd file at `~\AppData\Roaming\Python\Python37\site-packages\cv2`.

Thank for the help

**Comment by FernandoCalatayud** [comment]
> Sounds like you have a conflicting / another cv2 installation somewhere in your current environment. Create a fresh virtualenv and try to install `opencv-python` there. Also check that there is a cv2 .pyd file at `~\AppData\Roaming\Python\Python37\site-packages\cv2`.

Hi, i don't have the cv2.pyd ? I'm using conda env, and i already install all dependencies in a fresh env, but i still get the same error: ModuleNotFoundError: No module named 'cv2.cv2'

**Comment by rraghav476** [comment]
> > Sounds like you have a conflicting / another cv2 installation somewhere in your current environment. Create a fresh virtualenv and try to install `opencv-python` there. Also check that there is a cv2 .pyd file at `~\AppData\Roaming\Python\Python37\site-packages\cv2`.
> 
> Hi, i don't have the cv2.pyd ? I'm using conda env, and i already install all dependencies in a fresh env, but i still get the same error: ModuleNotFoundError: No module named 'cv2.cv2'

Hi i am also face same issue can u help me

**Comment by JohnieBraaf** [comment]
I had the same issue, turned out I was still in the source directory of opencv-python (which has cv2 dir as well).
Move to another dir, start python and import cv2.

**Comment by uniquecolin** [comment]
I had the same issue, is there anyone shed some light on this？My env is macos 10.15.7, python 3.9, pip 20.2.4, opencv-python 4.4.0.46

**Comment by iKap** [comment]
same issue, python 3.8.1

ModuleNotFoundError: No module named 'cv2.cv2'

HELP

**Comment by PierreLouisT** [comment]
Hi ! I got the same problem as well. 
I installed a conda environnement to use pypy3.7. I am using python 3.7. I used the line command in my environnement 
 _pypy3 -m pip install opencv-python_

It worked very well, however when I launched a pypy script which import cv2, I get the same issue than you guys : 
![image](https://user-images.githubusercontent.com/96771900/155295081-427defcc-7d23-4c13-a678-d2ec7287477b.png)

I tried to uninstall and reinstall with pip adapted for pypy, the same command as above. With python, it works without problem but pypy3 shows issues with opencv
